styles/base.css

```diff
@@ -33,6 +32,12 @@ body {
     }
 }
 
+@media (min-width: 1920px) {
+    .container-fluid {
+        max-width: 1320px;
+    }
+}
```

Comment:
I got the 1920px width breakpoint suggestion from CSS-Tricks: https://css-tricks.com/optimizing-large-scale-displays/#aa-build-with-relative-units They have breakpoints at 1920px, 2560px. Bootstrap breakpoints end with the largest at 1400px.

I got the 1320px width from Bootstrap: https://getbootstrap.com/docs/5.2/layout/containers/ It's the max-width for containers of screens up to 1400px.

Reply:
@thekaveman Can you try this on your machine? Does 1320px feel like a good width, or should it be wider?